### PR TITLE
event form

### PR DESCRIPTION
**Pull request by aring97**
git fetch --all
git checkout AR-eventForm
confirm that the event form is to the groups liking

### CODE REVIEW

**Review by mswinkels09 — COMMENTED** [review]
Looks great!! The only thing I see is that the form renders right at page load. Not sure if we wanted it to show up only when the new event button is pressed.

File: src/scripts/events/EventForm.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <p>Location of event:</p>
+    <input type="text" class="event--form__input" id="eventLocation">
+    <button id="event--form__close" class="close">Close event form</button>
+    <button id="create--event">submit</button>
+    </div>
+    `
+}
+
+eventHub.addEventListener("click", event=>{
+    const mobal=document.querySelector(".popup-container")
```
Comment (mgjeffries) [inline src/scripts/events/EventForm.js]: This element is a `modal` not a `mobal` please rename.

File: src/scripts/Nutshell.js [hunk 2]
```diff
@@ -6,6 +7,8 @@ import "./friends/FriendForm.js"
 
 
 export const Nutshell = () => {
+    // Render all your UI components here
+    eventForm()
```
Comment (mgjeffries) [inline src/scripts/Nutshell.js:11]: As @mswinkels09 mentioned, the event form should not display on page load.

File: src/styles/main.css [hunk 3]
```diff
@@ -1 +1,6 @@
-@import "./popup.css";
```
Comment (mgjeffries) [inline src/styles/main.css:1]: As @mswinkels09 mentioned, there is already styling for the popup on the master branch. Please don't delete the import for popup.css

**Review by mgjeffries — CHANGES_REQUESTED** [review]

**Review by mgjeffries — APPROVED** [review]
Looks great! Thanks for working with the classes for controlling visibility.

**Review by mswinkels09 — APPROVED** [review]
Looks great!!

**Pull request merged**